instat/dlgFactorLabels.vb

```diff
@@ -36,47 +36,48 @@ Public Class dlgViewFactorLabels
 
     Private Sub InitialiseDialog()
         'TODO: HElP ID
+
         ucrReceiverFactorColumns.SetParameter(New RParameter("x", 0))
         ucrReceiverFactorColumns.SetParameterIsRFunction()
         ucrReceiverFactorColumns.SetIncludedDataTypes({"factor", "ordered"})
         ucrReceiverFactorColumns.bForceAsDataFrame = True
         ucrReceiverFactorColumns.Selector = ucrSelectorViewFactorLabels
         ucrReceiverFactorColumns.SetMeAsReceiver()
 
-        ucrChkShowLabels.SetText("Show Variable Labels")
         ucrChkShowLabels.SetParameter(New RParameter("show.labels", 1))
+        ucrChkShowLabels.SetText("Show Variable Labels")

```

Comment:
My main change is very pedantic where I've started to put "New RParameter" first